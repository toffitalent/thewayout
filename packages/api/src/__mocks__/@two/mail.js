const send = jest.fn().mockResolvedValue(undefined);

module.exports = {
  send,
};
