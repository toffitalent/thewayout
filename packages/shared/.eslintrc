{
  "settings": {
    "import/resolver": {
      "typescript": {
        "project": ["tsconfig.json", "packages/shared/tsconfig.json"]
      }
    }
  },
  "rules": {
    "no-param-reassign": "off",
    "react/react-in-jsx-scope": "off"
  }
}
