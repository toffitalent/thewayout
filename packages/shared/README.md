# @two/shared

Copyright (c) 2023-Present The Way Out, Inc. All Rights Reserved.

## Development

This package contains code shared between the [api](../api/README.md), [app](../app/README.md), and [web](../web/README.md) packages and is not intended to run in a standalone manner. See the READMEs of those packages for information on starting a development environment.

## Testing

To run all component and unit tests and get coverage reports, simply run `yarn workspace @dlatwobs/shared test`.
