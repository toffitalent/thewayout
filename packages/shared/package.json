{
  "name": "@two/shared",
  "version": "1.0.0",
  "author": "Disruptive Labs",
  "license": "UNLICENSED",
  "private": true,
  "main": "build/index.js",
  "types": "build/index.d.ts",
  "scripts": {
    "build": "tsc -b tsconfig.build.json",
    "clean": "rimraf build coverage",
    "test": "jest --coverage",
    "type-check": "tsc"
  },
  "dependencies": {
    "@disruptive-labs/ui": "file:../../vendor/@disruptive-labs/ui",
    "axios": "^0.27.2",
    "react": "^18.2.0",
    "react-dom": "^18.2.0"
  },
  "devDependencies": {
    "@testing-library/dom": "^9.3.1",
    "@testing-library/jest-dom": "^5.16.5",
    "@testing-library/react": "^14.0.0",
    "@testing-library/user-event": "^14.4.3",
    "@types/jest": "^29.5.2",
    "@types/react": "^18.2.14",
    "babel-jest": "^29.5.0",
    "identity-obj-proxy": "^3.0.0",
    "jest": "^29.5.0",
    "jest-environment-jsdom": "^29.5.0",
    "jest-junit": "^16.0.0",
    "react-test-renderer": "^18.2.0",
    "rimraf": "^3.0.2",
    "sass": "^1.63.6",
    "typescript": "^5.2.2"
  },
  "jest": {
    "clearMocks": true,
    "collectCoverageFrom": [
      "<rootDir>/src/**/*.{ts,tsx}"
    ],
    "globalSetup": "<rootDir>/test/setup.js",
    "moduleNameMapper": {
      "^.+\\.module\\.(css|sass|scss)$": "identity-obj-proxy",
      "^.+\\.(css|sass|scss)$": "<rootDir>/test/styleMock.js",
      "^.+\\.(jpg|jpeg|png|gif|webp|svg)$": "<rootDir>/test/fileMock.js",
      "^@test$": "<rootDir>/test/index.tsx"
    },
    "setupFilesAfterEnv": [
      "<rootDir>/test/jest.test.ts"
    ],
    "testEnvironment": "jsdom",
    "testPathIgnorePatterns": [
      "<rootDir>/node_modules/",
      "<rootDir>/test/"
    ],
    "transform": {
      "^.+\\.(js|jsx|ts|tsx)$": [
        "babel-jest",
        {
          "presets": [
            "next/babel"
          ]
        }
      ]
    },
    "transformIgnorePatterns": [
      "node_modules/(?!(@?disruptive-labs.*|@popperjs/core|es-cookie)/)"
    ]
  },
  "jest-junit": {
    "outputDirectory": "./artifacts/jest",
    "outputName": "results.xml"
  }
}
