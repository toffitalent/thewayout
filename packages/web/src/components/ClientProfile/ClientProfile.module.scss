@use '@styles/mixins' as *;
@use '@styles/theme';

.rightColumn {
  padding-top: 4rem;

  @include mq(md, min) {
    padding-left: 2.5rem;
    padding-top: 2rem;
  }
}
