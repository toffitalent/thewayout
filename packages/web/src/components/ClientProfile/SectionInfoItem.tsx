import { Box, Flex, FlexComponentProps, Text } from '@disruptive-labs/ui';

interface SectionInfoItemProps extends FlexComponentProps {
  icon: any;
  title: string;
  text1: string;
  text2: string;
  text3?: string;
  description?: string;
}

export const SectionInfoItem = ({
  icon,
  title,
  text1,
  text2,
  text3,
  description,
  ...props
}: SectionInfoItemProps) => (
  <Flex direction="row" {...props}>
    <Box color="primary" mr={5} display="flex">
      {icon}
    </Box>
    <Box>
      <Text fontWeight="700">{title}</Text>
      <Text fontSize="sm">{text1}</Text>
      <Text fontSize="sm" color="grey.500">
        {text2}
      </Text>
      {text3 && (
        <Text fontSize="sm" color="grey.500">
          {text3}
        </Text>
      )}
      <Text fontSize="sm" pt={2}>
        {description}
      </Text>
    </Box>
  </Flex>
);
