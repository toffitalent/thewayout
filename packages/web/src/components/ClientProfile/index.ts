export * from './ClientProfile';
export * from './SectionInfoItem';
