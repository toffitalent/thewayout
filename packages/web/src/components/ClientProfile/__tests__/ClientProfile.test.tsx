import { UserType } from '@two/shared';
import { fixtures, render } from '@test';
import { ClientProfile } from '../ClientProfile';

describe('ClientProfile', () => {
  test('renders correctly for employer', () => {
    const { container, queryByText, queryByTestId } = render(
      <ClientProfile client={fixtures.client} type={UserType.Employer} />,
    );
    expect(queryByText('Offense History')).not.toBeInTheDocument();
    expect(queryByText('Support Needed')).not.toBeInTheDocument();
    expect(queryByTestId('cloaked')).not.toBeInTheDocument();
    expect(container).toMatchSnapshot();
  });

  test('renders correctly for rsp', () => {
    const { container, queryByText, queryByTestId } = render(
      <ClientProfile client={fixtures.client} type={UserType.Rsp} />,
    );
    expect(queryByText('Offense History')).toBeInTheDocument();
    expect(queryByText('Support Needed')).toBeInTheDocument();
    expect(queryByTestId('cloaked')).not.toBeInTheDocument();
    expect(container).toMatchSnapshot();
  });

  test('renders correctly for client', () => {
    const { container, queryByText, queryByTestId } = render(
      <ClientProfile client={fixtures.client} type={UserType.Client} />,
    );
    expect(queryByText('Offense History')).toBeInTheDocument();
    expect(queryByText('Support Needed')).toBeInTheDocument();
    expect(queryByTestId('cloaked')).toBeInTheDocument();
    expect(container).toMatchSnapshot();
  });
});
