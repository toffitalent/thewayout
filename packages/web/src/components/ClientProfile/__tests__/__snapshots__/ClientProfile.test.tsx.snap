// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ClientProfile renders correctly for client 1`] = `
<div>
  <div
    class="container"
  >
    <div
      class="item bp-xs-12 bp-md-4 flexDirection-column display-flex"
      style="gap: 2rem;"
    >
      <div
        class="avatar s-2xl"
      >
        <div
          aria-label="Test User"
          class="initials"
        >
          TU
        </div>
      </div>
      <div
        class=""
      >
        <div
          class="alignItems-center flexDirection-row"
        >
          <p
            class="text fontSize-3xl fontWeight-700"
          >
            Test User
          </p>
          <svg
            aria-hidden="true"
            class="icon color-primary width-6 height-6 marginLeft-3"
            cursor="pointer"
            data-testid="cloaked"
            focusable="false"
            stroke-width="2"
            viewBox="0 0 24 24"
          >
            <path
              d="M0 0h24v24H0z"
              stroke="none"
            />
            <path
              d="M10.585 10.587a2 2 0 002.829 2.828M16.681 16.673A8.717 8.717 0 0112 18c-3.6 0-6.6-2-9-6 1.272-2.12 2.712-3.678 4.32-4.674m2.86-1.146A9.055 9.055 0 0112 6c3.6 0 6.6 2 9 6-.666 1.11-1.379 2.067-2.138 2.87M3 3l18 18"
            />
          </svg>
        </div>
        <div
          class="readMore inline"
        >
          <p
            class="text numLines-2 whiteSpace-pre-line"
          >
            personal summary
          </p>
        </div>
      </div>
      <div
        class=""
      >
        <div
          class="container flexDirection-row"
        >
          <div
            class="item bp-xs-12 bp-md-8 alignItems-center flexDirection-row"
          >
            <svg
              aria-hidden="true"
              class="icon"
              cursor="pointer"
              data-testid="icon-up"
              focusable="false"
              stroke-width="2"
              viewBox="0 0 24 24"
            >
              <path
                d="M0 0h24v24H0z"
                stroke="none"
              />
              <path
                d="M6 15l6-6 6 6"
              />
            </svg>
            <p
              class="text marginLeft-2 fontSize-xl fontWeight-700"
            >
              Personal Details
            </p>
            <svg
              aria-hidden="true"
              class="icon color-primary width-6 height-6 marginLeft-3"
              cursor="pointer"
              focusable="false"
              stroke-width="2"
              viewBox="0 0 24 24"
            >
              <path
                d="M0 0h24v24H0z"
                stroke="none"
              />
              <path
                d="M4 20h4L18.5 9.5a2.828 2.828 0 10-4-4L4 16v4M13.5 6.5l4 4"
              />
            </svg>
          </div>
        </div>
        <div
          class="paddingTop-4"
        >
          <div
            class="container spacing-2"
          >
            <div
              class="item flexDirection-row width-full"
            >
              <p
                class="text color-grey-700 width-1/2"
              >
                Email
              </p>
              <p
                class="text width-1/2"
                style="overflow-wrap: break-word;"
              >
                test@example.com
              </p>
            </div>
            <div
              class="item flexDirection-row width-full"
            >
              <p
                class="text color-grey-700 width-1/2"
              >
                Address
              </p>
              <p
                class="text width-1/2"
                style="overflow-wrap: break-word;"
              >
                111 Street, 
 Los Angeles CA 90887
              </p>
            </div>
          </div>
        </div>
      </div>
      <div
        class=""
      >
        <div
          class="container flexDirection-row"
        >
          <div
            class="item bp-xs-12 bp-md-8 alignItems-center flexDirection-row"
          >
            <svg
              aria-hidden="true"
              class="icon"
              cursor="pointer"
              data-testid="icon-up"
              focusable="false"
              stroke-width="2"
              viewBox="0 0 24 24"
            >
              <path
                d="M0 0h24v24H0z"
                stroke="none"
              />
              <path
                d="M6 15l6-6 6 6"
              />
            </svg>
            <p
              class="text marginLeft-2 fontSize-xl fontWeight-700"
            >
              Offense History
            </p>
            <svg
              aria-hidden="true"
              class="icon color-primary width-6 height-6 marginLeft-3"
              cursor="pointer"
              focusable="false"
              stroke-width="2"
              viewBox="0 0 24 24"
            >
              <path
                d="M0 0h24v24H0z"
                stroke="none"
              />
              <path
                d="M4 20h4L18.5 9.5a2.828 2.828 0 10-4-4L4 16v4M13.5 6.5l4 4"
              />
            </svg>
          </div>
        </div>
        <div
          class="paddingTop-4"
        >
          <div
            class="container spacing-2"
          >
            <div
              class="item flexDirection-row width-full"
            >
              <p
                class="text color-grey-700 width-1/2"
              >
                Justice Status
              </p>
              <p
                class="text width-1/2"
                style="overflow-wrap: break-word;"
              >
                Free World
              </p>
            </div>
            <div
              class="item flexDirection-row width-full"
            >
              <p
                class="text color-grey-700 width-1/2"
              >
                Nature of Offense
              </p>
              <p
                class="text width-1/2"
                style="overflow-wrap: break-word;"
              >
                Arson
              </p>
            </div>
            <div
              class="item flexDirection-row width-full"
            >
              <p
                class="text color-grey-700 width-1/2"
              >
                Time Served
              </p>
              <p
                class="text width-1/2"
                style="overflow-wrap: break-word;"
              >
                Less than 1 year
              </p>
            </div>
            <div
              class="item flexDirection-row width-full"
            >
              <p
                class="text color-grey-700 width-1/2"
              >
                Completed Sentence
              </p>
              <p
                class="text width-1/2"
                style="overflow-wrap: break-word;"
              >
                Less than 1 year
              </p>
            </div>
            <div
              class="item flexDirection-row width-full"
            >
              <p
                class="text color-grey-700 width-1/2"
              >
                State or Federal
              </p>
              <p
                class="text width-1/2"
                style="overflow-wrap: break-word;"
              >
                Federal
              </p>
            </div>
          </div>
        </div>
      </div>
      <div
        class=""
      >
        <div
          class="container flexDirection-row"
        >
          <div
            class="item bp-xs-12 bp-md-8 alignItems-center flexDirection-row"
          >
            <svg
              aria-hidden="true"
              class="icon"
              cursor="pointer"
              data-testid="icon-up"
              focusable="false"
              stroke-width="2"
              viewBox="0 0 24 24"
            >
              <path
                d="M0 0h24v24H0z"
                stroke="none"
              />
              <path
                d="M6 15l6-6 6 6"
              />
            </svg>
            <p
              class="text marginLeft-2 fontSize-xl fontWeight-700"
            >
              Languages
            </p>
            <svg
              aria-hidden="true"
              class="icon color-primary width-6 height-6 marginLeft-3"
              cursor="pointer"
              focusable="false"
              stroke-width="2"
              viewBox="0 0 24 24"
            >
              <path
                d="M0 0h24v24H0z"
                stroke="none"
              />
              <path
                d="M3 12a9 9 0 1018 0 9 9 0 00-18 0M9 12h6M12 9v6"
              />
            </svg>
          </div>
        </div>
        <div
          class="paddingTop-4"
        >
          <div
            class="container spacing-1"
          />
        </div>
      </div>
      <div
        class=""
      >
        <div
          class="container flexDirection-row"
        >
          <div
            class="item bp-xs-12 bp-md-8 alignItems-center flexDirection-row"
          >
            <svg
              aria-hidden="true"
              class="icon"
              cursor="pointer"
              data-testid="icon-up"
              focusable="false"
              stroke-width="2"
              viewBox="0 0 24 24"
            >
              <path
                d="M0 0h24v24H0z"
                stroke="none"
              />
              <path
                d="M6 15l6-6 6 6"
              />
            </svg>
            <p
              class="text marginLeft-2 fontSize-xl fontWeight-700"
            >
              Personal Strengths
            </p>
            <svg
              aria-hidden="true"
              class="icon color-primary width-6 height-6 marginLeft-3"
              cursor="pointer"
              focusable="false"
              stroke-width="2"
              viewBox="0 0 24 24"
            >
              <path
                d="M0 0h24v24H0z"
                stroke="none"
              />
              <path
                d="M3 12a9 9 0 1018 0 9 9 0 00-18 0M9 12h6M12 9v6"
              />
            </svg>
          </div>
        </div>
        <div
          class="paddingTop-4"
        >
          <div
            class="container spacing-1"
          />
        </div>
      </div>
      <div
        class=""
      >
        <div
          class="container flexDirection-row"
        >
          <div
            class="item bp-xs-12 bp-md-8 alignItems-center flexDirection-row"
          >
            <svg
              aria-hidden="true"
              class="icon"
              cursor="pointer"
              data-testid="icon-up"
              focusable="false"
              stroke-width="2"
              viewBox="0 0 24 24"
            >
              <path
                d="M0 0h24v24H0z"
                stroke="none"
              />
              <path
                d="M6 15l6-6 6 6"
              />
            </svg>
            <p
              class="text marginLeft-2 fontSize-xl fontWeight-700"
            >
              Skills
            </p>
            <svg
              aria-hidden="true"
              class="icon color-primary width-6 height-6 marginLeft-3"
              cursor="pointer"
              focusable="false"
              stroke-width="2"
              viewBox="0 0 24 24"
            >
              <path
                d="M0 0h24v24H0z"
                stroke="none"
              />
              <path
                d="M3 12a9 9 0 1018 0 9 9 0 00-18 0M9 12h6M12 9v6"
              />
            </svg>
          </div>
        </div>
        <div
          class="paddingTop-4"
        >
          <div
            class="container spacing-1"
          />
        </div>
      </div>
      <div
        class=""
      >
        <div
          class="container flexDirection-row"
        >
          <div
            class="item bp-xs-12 bp-md-8 alignItems-center flexDirection-row"
          >
            <svg
              aria-hidden="true"
              class="icon"
              cursor="pointer"
              data-testid="icon-up"
              focusable="false"
              stroke-width="2"
              viewBox="0 0 24 24"
            >
              <path
                d="M0 0h24v24H0z"
                stroke="none"
              />
              <path
                d="M6 15l6-6 6 6"
              />
            </svg>
            <p
              class="text marginLeft-2 fontSize-xl fontWeight-700"
            >
              Support Needed
            </p>
          </div>
        </div>
        <div
          class="paddingTop-4"
        >
          <div
            class="container spacing-1"
          >
            <div
              class="item"
            >
              <div
                class="badge borderColor-indigo-200 bgcolor-indigo-200 color-indigo-800 marginRight-2"
              >
                Resume
              </div>
            </div>
          </div>
        </div>
      </div>
    </div>
    <div
      class="rightColumn item bp-xs-12 bp-md-8"
    >
      <div
        class="marginBottom-10"
      >
        <div
          class="alignItems-center flexDirection-row"
        >
          <p
            class="text fontSize-xl fontWeight-700"
          >
            Work Experience
          </p>
          <svg
            aria-hidden="true"
            class="icon color-primary width-6 height-6 marginLeft-3"
            cursor="pointer"
            focusable="false"
            stroke-width="2"
            viewBox="0 0 24 24"
          >
            <path
              d="M0 0h24v24H0z"
              stroke="none"
            />
            <path
              d="M4 20h4L18.5 9.5a2.828 2.828 0 10-4-4L4 16v4M13.5 6.5l4 4"
            />
          </svg>
        </div>
      </div>
      <div
        class="marginBottom-10"
      >
        <div
          class="alignItems-center flexDirection-row"
        >
          <p
            class="text fontSize-xl fontWeight-700"
          >
            Education
          </p>
          <svg
            aria-hidden="true"
            class="icon color-primary width-6 height-6 marginLeft-3"
            cursor="pointer"
            focusable="false"
            stroke-width="2"
            viewBox="0 0 24 24"
          >
            <path
              d="M0 0h24v24H0z"
              stroke="none"
            />
            <path
              d="M4 20h4L18.5 9.5a2.828 2.828 0 10-4-4L4 16v4M13.5 6.5l4 4"
            />
          </svg>
        </div>
      </div>
      <div
        class=""
      >
        <div
          class="alignItems-center flexDirection-row"
        >
          <p
            class="text fontSize-xl fontWeight-700"
          >
            Licenses / Certifications
          </p>
          <svg
            aria-hidden="true"
            class="icon color-primary width-6 height-6 marginLeft-3"
            cursor="pointer"
            focusable="false"
            stroke-width="2"
            viewBox="0 0 24 24"
          >
            <path
              d="M0 0h24v24H0z"
              stroke="none"
            />
            <path
              d="M4 20h4L18.5 9.5a2.828 2.828 0 10-4-4L4 16v4M13.5 6.5l4 4"
            />
          </svg>
        </div>
      </div>
    </div>
  </div>
</div>
`;

exports[`ClientProfile renders correctly for employer 1`] = `
<div>
  <div
    class="container"
  >
    <div
      class="item bp-xs-12 bp-md-4 flexDirection-column display-flex"
      style="gap: 2rem;"
    >
      <div
        class="avatar s-2xl"
      >
        <div
          aria-label="Test User"
          class="initials"
        >
          TU
        </div>
      </div>
      <div
        class=""
      >
        <div
          class="alignItems-center flexDirection-row"
        >
          <p
            class="text fontSize-3xl fontWeight-700"
          >
            Test User
          </p>
        </div>
        <div
          class="readMore inline"
        >
          <p
            class="text numLines-2 whiteSpace-pre-line"
          >
            personal summary
          </p>
        </div>
      </div>
      <div
        class=""
      >
        <div
          class="container flexDirection-row"
        >
          <div
            class="item bp-xs-12 bp-md-8 alignItems-center flexDirection-row"
          >
            <svg
              aria-hidden="true"
              class="icon"
              cursor="pointer"
              data-testid="icon-up"
              focusable="false"
              stroke-width="2"
              viewBox="0 0 24 24"
            >
              <path
                d="M0 0h24v24H0z"
                stroke="none"
              />
              <path
                d="M6 15l6-6 6 6"
              />
            </svg>
            <p
              class="text marginLeft-2 fontSize-xl fontWeight-700"
            >
              Client Details
            </p>
          </div>
        </div>
        <div
          class="paddingTop-4"
        >
          <div
            class="container spacing-2"
          >
            <div
              class="item flexDirection-row width-full"
            >
              <p
                class="text color-grey-700 width-1/2"
              >
                Email
              </p>
              <p
                class="text width-1/2"
                style="overflow-wrap: break-word;"
              >
                test@example.com
              </p>
            </div>
            <div
              class="item flexDirection-row width-full"
            >
              <p
                class="text color-grey-700 width-1/2"
              >
                Address
              </p>
              <p
                class="text width-1/2"
                style="overflow-wrap: break-word;"
              >
                111 Street, 
 Los Angeles CA 90887
              </p>
            </div>
          </div>
        </div>
      </div>
      <div
        class=""
      >
        <div
          class="container flexDirection-row"
        >
          <div
            class="item bp-xs-12 bp-md-8 alignItems-center flexDirection-row"
          >
            <svg
              aria-hidden="true"
              class="icon"
              cursor="pointer"
              data-testid="icon-up"
              focusable="false"
              stroke-width="2"
              viewBox="0 0 24 24"
            >
              <path
                d="M0 0h24v24H0z"
                stroke="none"
              />
              <path
                d="M6 15l6-6 6 6"
              />
            </svg>
            <p
              class="text marginLeft-2 fontSize-xl fontWeight-700"
            >
              Languages
            </p>
          </div>
        </div>
        <div
          class="paddingTop-4"
        >
          <div
            class="container spacing-1"
          />
        </div>
      </div>
      <div
        class=""
      >
        <div
          class="container flexDirection-row"
        >
          <div
            class="item bp-xs-12 bp-md-8 alignItems-center flexDirection-row"
          >
            <svg
              aria-hidden="true"
              class="icon"
              cursor="pointer"
              data-testid="icon-up"
              focusable="false"
              stroke-width="2"
              viewBox="0 0 24 24"
            >
              <path
                d="M0 0h24v24H0z"
                stroke="none"
              />
              <path
                d="M6 15l6-6 6 6"
              />
            </svg>
            <p
              class="text marginLeft-2 fontSize-xl fontWeight-700"
            >
              Personal Strengths
            </p>
          </div>
        </div>
        <div
          class="paddingTop-4"
        >
          <div
            class="container spacing-1"
          />
        </div>
      </div>
      <div
        class=""
      >
        <div
          class="container flexDirection-row"
        >
          <div
            class="item bp-xs-12 bp-md-8 alignItems-center flexDirection-row"
          >
            <svg
              aria-hidden="true"
              class="icon"
              cursor="pointer"
              data-testid="icon-up"
              focusable="false"
              stroke-width="2"
              viewBox="0 0 24 24"
            >
              <path
                d="M0 0h24v24H0z"
                stroke="none"
              />
              <path
                d="M6 15l6-6 6 6"
              />
            </svg>
            <p
              class="text marginLeft-2 fontSize-xl fontWeight-700"
            >
              Skills
            </p>
          </div>
        </div>
        <div
          class="paddingTop-4"
        >
          <div
            class="container spacing-1"
          />
        </div>
      </div>
    </div>
    <div
      class="rightColumn item bp-xs-12 bp-md-8"
    >
      <div
        class="marginBottom-10"
      >
        <div
          class="alignItems-center flexDirection-row"
        >
          <p
            class="text fontSize-xl fontWeight-700"
          >
            Work Experience
          </p>
        </div>
      </div>
      <div
        class="marginBottom-10"
      >
        <div
          class="alignItems-center flexDirection-row"
        >
          <p
            class="text fontSize-xl fontWeight-700"
          >
            Education
          </p>
        </div>
      </div>
      <div
        class=""
      >
        <div
          class="alignItems-center flexDirection-row"
        >
          <p
            class="text fontSize-xl fontWeight-700"
          >
            Licenses / Certifications
          </p>
        </div>
      </div>
    </div>
  </div>
</div>
`;

exports[`ClientProfile renders correctly for rsp 1`] = `
<div>
  <div
    class="container"
  >
    <div
      class="item bp-xs-12 bp-md-4 flexDirection-column display-flex"
      style="gap: 2rem;"
    >
      <div
        class="avatar s-2xl"
      >
        <div
          aria-label="Test User"
          class="initials"
        >
          TU
        </div>
      </div>
      <div
        class=""
      >
        <div
          class="alignItems-center flexDirection-row"
        >
          <p
            class="text fontSize-3xl fontWeight-700"
          >
            Test User
          </p>
        </div>
        <div
          class="readMore inline"
        >
          <p
            class="text numLines-2 whiteSpace-pre-line"
          >
            personal summary
          </p>
        </div>
      </div>
      <div
        class=""
      >
        <div
          class="container flexDirection-row"
        >
          <div
            class="item bp-xs-12 bp-md-8 alignItems-center flexDirection-row"
          >
            <svg
              aria-hidden="true"
              class="icon"
              cursor="pointer"
              data-testid="icon-up"
              focusable="false"
              stroke-width="2"
              viewBox="0 0 24 24"
            >
              <path
                d="M0 0h24v24H0z"
                stroke="none"
              />
              <path
                d="M6 15l6-6 6 6"
              />
            </svg>
            <p
              class="text marginLeft-2 fontSize-xl fontWeight-700"
            >
              Client Details
            </p>
          </div>
        </div>
        <div
          class="paddingTop-4"
        >
          <div
            class="container spacing-2"
          >
            <div
              class="item flexDirection-row width-full"
            >
              <p
                class="text color-grey-700 width-1/2"
              >
                Email
              </p>
              <p
                class="text width-1/2"
                style="overflow-wrap: break-word;"
              >
                test@example.com
              </p>
            </div>
            <div
              class="item flexDirection-row width-full"
            >
              <p
                class="text color-grey-700 width-1/2"
              >
                Address
              </p>
              <p
                class="text width-1/2"
                style="overflow-wrap: break-word;"
              >
                111 Street, 
 Los Angeles CA 90887
              </p>
            </div>
          </div>
        </div>
      </div>
      <div
        class=""
      >
        <div
          class="container flexDirection-row"
        >
          <div
            class="item bp-xs-12 bp-md-8 alignItems-center flexDirection-row"
          >
            <svg
              aria-hidden="true"
              class="icon"
              cursor="pointer"
              data-testid="icon-up"
              focusable="false"
              stroke-width="2"
              viewBox="0 0 24 24"
            >
              <path
                d="M0 0h24v24H0z"
                stroke="none"
              />
              <path
                d="M6 15l6-6 6 6"
              />
            </svg>
            <p
              class="text marginLeft-2 fontSize-xl fontWeight-700"
            >
              Offense History
            </p>
          </div>
        </div>
        <div
          class="paddingTop-4"
        >
          <div
            class="container spacing-2"
          >
            <div
              class="item flexDirection-row width-full"
            >
              <p
                class="text color-grey-700 width-1/2"
              >
                Justice Status
              </p>
              <p
                class="text width-1/2"
                style="overflow-wrap: break-word;"
              >
                Free World
              </p>
            </div>
            <div
              class="item flexDirection-row width-full"
            >
              <p
                class="text color-grey-700 width-1/2"
              >
                Nature of Offense
              </p>
              <p
                class="text width-1/2"
                style="overflow-wrap: break-word;"
              >
                Arson
              </p>
            </div>
            <div
              class="item flexDirection-row width-full"
            >
              <p
                class="text color-grey-700 width-1/2"
              >
                Time Served
              </p>
              <p
                class="text width-1/2"
                style="overflow-wrap: break-word;"
              >
                Less than 1 year
              </p>
            </div>
            <div
              class="item flexDirection-row width-full"
            >
              <p
                class="text color-grey-700 width-1/2"
              >
                Completed Sentence
              </p>
              <p
                class="text width-1/2"
                style="overflow-wrap: break-word;"
              >
                Less than 1 year
              </p>
            </div>
            <div
              class="item flexDirection-row width-full"
            >
              <p
                class="text color-grey-700 width-1/2"
              >
                State or Federal
              </p>
              <p
                class="text width-1/2"
                style="overflow-wrap: break-word;"
              >
                Federal
              </p>
            </div>
          </div>
        </div>
      </div>
      <div
        class=""
      >
        <div
          class="container flexDirection-row"
        >
          <div
            class="item bp-xs-12 bp-md-8 alignItems-center flexDirection-row"
          >
            <svg
              aria-hidden="true"
              class="icon"
              cursor="pointer"
              data-testid="icon-up"
              focusable="false"
              stroke-width="2"
              viewBox="0 0 24 24"
            >
              <path
                d="M0 0h24v24H0z"
                stroke="none"
              />
              <path
                d="M6 15l6-6 6 6"
              />
            </svg>
            <p
              class="text marginLeft-2 fontSize-xl fontWeight-700"
            >
              Languages
            </p>
          </div>
        </div>
        <div
          class="paddingTop-4"
        >
          <div
            class="container spacing-1"
          />
        </div>
      </div>
      <div
        class=""
      >
        <div
          class="container flexDirection-row"
        >
          <div
            class="item bp-xs-12 bp-md-8 alignItems-center flexDirection-row"
          >
            <svg
              aria-hidden="true"
              class="icon"
              cursor="pointer"
              data-testid="icon-up"
              focusable="false"
              stroke-width="2"
              viewBox="0 0 24 24"
            >
              <path
                d="M0 0h24v24H0z"
                stroke="none"
              />
              <path
                d="M6 15l6-6 6 6"
              />
            </svg>
            <p
              class="text marginLeft-2 fontSize-xl fontWeight-700"
            >
              Personal Strengths
            </p>
          </div>
        </div>
        <div
          class="paddingTop-4"
        >
          <div
            class="container spacing-1"
          />
        </div>
      </div>
      <div
        class=""
      >
        <div
          class="container flexDirection-row"
        >
          <div
            class="item bp-xs-12 bp-md-8 alignItems-center flexDirection-row"
          >
            <svg
              aria-hidden="true"
              class="icon"
              cursor="pointer"
              data-testid="icon-up"
              focusable="false"
              stroke-width="2"
              viewBox="0 0 24 24"
            >
              <path
                d="M0 0h24v24H0z"
                stroke="none"
              />
              <path
                d="M6 15l6-6 6 6"
              />
            </svg>
            <p
              class="text marginLeft-2 fontSize-xl fontWeight-700"
            >
              Skills
            </p>
          </div>
        </div>
        <div
          class="paddingTop-4"
        >
          <div
            class="container spacing-1"
          />
        </div>
      </div>
      <div
        class=""
      >
        <div
          class="container flexDirection-row"
        >
          <div
            class="item bp-xs-12 bp-md-8 alignItems-center flexDirection-row"
          >
            <svg
              aria-hidden="true"
              class="icon"
              cursor="pointer"
              data-testid="icon-up"
              focusable="false"
              stroke-width="2"
              viewBox="0 0 24 24"
            >
              <path
                d="M0 0h24v24H0z"
                stroke="none"
              />
              <path
                d="M6 15l6-6 6 6"
              />
            </svg>
            <p
              class="text marginLeft-2 fontSize-xl fontWeight-700"
            >
              Support Needed
            </p>
          </div>
        </div>
        <div
          class="paddingTop-4"
        >
          <div
            class="container spacing-1"
          >
            <div
              class="item"
            >
              <div
                class="badge borderColor-indigo-200 bgcolor-indigo-200 color-indigo-800 marginRight-2"
              >
                Resume
              </div>
            </div>
          </div>
        </div>
      </div>
    </div>
    <div
      class="rightColumn item bp-xs-12 bp-md-8"
    >
      <div
        class="marginBottom-10"
      >
        <div
          class="alignItems-center flexDirection-row"
        >
          <p
            class="text fontSize-xl fontWeight-700"
          >
            Work Experience
          </p>
        </div>
      </div>
      <div
        class="marginBottom-10"
      >
        <div
          class="alignItems-center flexDirection-row"
        >
          <p
            class="text fontSize-xl fontWeight-700"
          >
            Education
          </p>
        </div>
      </div>
      <div
        class=""
      >
        <div
          class="alignItems-center flexDirection-row"
        >
          <p
            class="text fontSize-xl fontWeight-700"
          >
            Licenses / Certifications
          </p>
        </div>
      </div>
    </div>
  </div>
</div>
`;
